src/libData/AccountData/Account.cpp

```diff
@@ -35,8 +35,105 @@ using namespace dev;
 
 using namespace Contract;
 
-Account::Account() {}
+// =======================================
+// AccountBase
 
+AccountBase::AccountBase(const uint128_t& balance, const uint64_t& nonce,
+                         const uint32_t& version)
+    : m_version(version),
```

Comment:
Build in type can just pass in by value, const uint32_t& version can change to uint32_t version